modules/ROOT/pages/webhooks.adoc

```diff
@@ -124,6 +124,25 @@ Sets the address to which documents are posted.
 
 --
 
+Options::
++
+--
+Property name: {configuration-schema-database--xref--eventhandlers-doc-changed-options}
+
+The options object configures additional webhook behavior. 
+
+Available options include:
+
+* `winning_rev_only`
```

Comment:
It is somewhat confusing because there's only one available option and the only applicable value is `"winning_rev_only": true`

I wonder if is more idiomatic to just say  `"winning_rev_only": true` like `"enable_delta_sync": true ` https://docs.couchbase.com/sync-gateway/current/sync-inter-syncgateway-overview.html#delta-sync